src/pages/academy/grading/subcomponents/GradingWorkspace.tsx

```diff
@@ -238,6 +257,16 @@ const GradingWorkspace: React.FC<GradingWorkspaceProps> = props => {
       if (!editorValue) {
         editorValue = questionData.solutionTemplate!;
       }
+      if (
+        editorValue.trim() === questionData.solutionTemplate?.trim() ||
+        editorValue.replace(whitespaceRegex, '') ===
```

Comment:
Might be better to use `replaceAll` instead of `replace` as it is safer (it will throw a `TypeError` if the regex passed in is not created with the `/g` flag).